pkg/services/publicdashboards/database/database.go

```diff
@@ -241,8 +241,9 @@ func (d *PublicDashboardStoreImpl) UpdatePublicDashboardConfig(ctx context.Conte
 			return err
 		}
 
-		_, err = sess.Exec("UPDATE dashboard_public SET is_enabled = ?, time_settings = ?, updated_by = ?, updated_at = ? WHERE uid = ?",
+		_, err = sess.Exec("UPDATE dashboard_public SET is_enabled = ?, annotations_enabled = ?, time_settings = ?, updated_by = ?, updated_at = ? WHERE uid = ?",
 			cmd.PublicDashboard.IsEnabled,
+			cmd.PublicDashboard.IsAnnotationsEnabled,
```

Comment:
Sorry for the nit. I think this should be "AnnotationsEnabled" not IsAnnotationsEnabled"

Reply:
Ohh I see what you mean, maybe this could be `hasAnnotationsEnabled` but I like to add a prefix `is` or `has` so you know just reading the name what type of variable/attribute it is.

Reply:
That's also a valid name. I think either is fine. Gonna stick with `annotationsEnabled` for now.